src/app/components/common/footer/footer.component.html

```diff
@@ -0,0 +1,3 @@
+<mat-toolbar color="secondary">
+  <span class="mat-small">@copyright 2020</span>
```

Comment:
can we add localization module in next PR

https://angular.io/guide/i18n#localizing-your-app